src/toil/lib/conversions.py

```diff
@@ -1,4 +1,8 @@
-"""Conversion utilities for mapping memory, disk, core declarations from strings to numbers and vice versa."""
+"""
+Conversion utilities for mapping memory, disk, core declarations from strings to numbers and vice versa.
+Conversions.py is now for general conversion functions as of July 13 2021
```

Comment:
Are you sure this sort of log-like documentation is more useful than just editing the comment to describe what is now true?

Reply:
No, that's more just a habit from class assignments, where they say document everything and then I don't document it in a seperate thing so i go back and see when I changed things cause some of the journals were meant to be chronological about issues, so in this scenario probably not